controllers/webspherelibertyapplication_controller.go

```diff
@@ -670,6 +672,14 @@ func (r *ReconcileWebSphereLiberty) Reconcile(ctx context.Context, request ctrl.
 	return r.ManageSuccess(common.StatusConditionTypeReconciled, instance)
 }
 
+func (r *ReconcileWebSphereLiberty) isWebSphereLibertyApplicationReady(ba common.BaseComponent) bool {
+	if r.CheckApplicationStatus(ba) == corev1.ConditionTrue {
```

Comment:
Wondering if the call to `CheckApplicationStatus` can be omitted as it's checked in ManageSuccess. The removal of old semeru instance may only happen in the next reconcile, which would probably be okay.